autonomy.py

```diff
@@ -23,11 +23,11 @@ def main() -> None:
 
     # Setup feeds for AR Tag and obstacle avoidance
     core.vision.feed_handler.add_feed(2, "artag", stream_video=core.vision.STREAM_FLAG)
-    core.vision.feed_handler.add_feed(3, "obstacle", stream_video=core.vision.STREAM_FLAG)
+    # core.vision.feed_handler.add_feed(3, "obstacle", stream_video=core.vision.STREAM_FLAG)
 
     # Create our two detection tasks
     loop.create_task(core.vision.ar_tag_detector.async_ar_tag_detector())
-    loop.create_task(core.vision.obstacle_avoidance.async_obstacle_detector())
+    # loop.create_task(core.vision.obstacle_avoidance.async_obstacle_detector())
```

Comment:
Consider uncommenting these lines if you want to run obstacle avoidance or explaining why it isn't being run.